### ISSUE TITLE
Error 1171: All parts of a PRIMARY KEY must be NOT NULL; if you need NULL in a key, use UNIQUE instead

### ISSUE THREAD

**Issue by hussainabrar** [phpmyadmin/phpmyadmin#19480]
### Describe the bug

I am running the gh-ost command first time and facing this error..

Commnd executed is : 
gh-ost --allow-on-master --assume-rbr --exact-rowcount --critical-load Threads_running=400 --critical-load-hibernate-seconds 60 --database database_name --max-load Threads_running=100 --nice-ratio 0.1 --chunk-size 5000 --ask-pass --table table_to_modify --user username --host hostname --postpone-cut-over-flag-file /tmp/gh-ost-postpone-coff --throttle-additional-flag-file /tmp/gh-ost-throttle_ghost.sbtest1 --alter 'MODIFY column_to_modify BIGINT NOT NULL AUTO_INCREMENT DEFAULT NULL' --verbose -execute 2>&1 | tee gh-ost.log

After this, running into an error : 

### To Reproduce

Ran below command :

Current table definition : 

CREATE TABLE `table_to_modify` (
  `**column_to_modify**` int NOT NULL AUTO_INCREMENT,
  `transaction_id` int NOT NULL,
  `name` varchar(50) COLLATE utf8mb4_general_ci NOT NULL,
  `start` datetime DEFAULT NULL,
  `stop` datetime DEFAULT NULL,
  `created_on` datetime NOT NULL,
  `row_stamp` datetime(3) DEFAULT CURRENT_TIMESTAMP(3) ON UPDATE CURRENT_TIMESTAMP(3),
  PRIMARY KEY (`transaction_sla_id`),
  UNIQUE KEY `date_observed_idx` (`transaction_id`,`name`)
) ENGINE=InnoDB AUTO_INCREMENT=3046722 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci

### Screenshots

If applicable, add screenshots to help explain the bug.

### Server configuration

- Operating system:
- Web server:
- Database version:
- PHP version:
- phpMyAdmin version:

### Client configuration

- Browser:
- Operating system:

### Additional context

Add any other context about the bug here.

**Comment by kamil-tekiela** [comment]
Is this related to phpMyAdmin?

**Comment by hussainabrar** [comment]
No, its related to Mysql..The issue is resolved by removing DEFAULT NULL from gh-ost command..

**Comment by kamil-tekiela** [comment]
Then why did you report it as a phpmyadmin bug? I am trying to understand what the issue is.
